app/javascript/components/pages/ProtectedShow.js

```diff
@@ -25,6 +26,7 @@ const ProtectedShow = ({
         {myOwnMedications?.map((DisplayMyMeds)=>{
         return (
               <>
+              <NavLink to={`/${DisplayMyMeds.user_id}/medications/${DisplayMyMeds.id}`}>
```

Comment:
Please be mindful of the naming conventions of parameters. Javascript parameters should be camelCased. Referring to Line 26